### PR TITLE
Added Simprints Identification to the library

### PR DESCRIPTION
**Pull request by issyzac**
Added implementations to include Simprints Identification process to the library

### CODE REVIEW

**Review by mahmud6390 — APPROVED** [review]
seems ok

File: sample/src/main/java/org/smartregister/simprint/sample/MainActivity.java [hunk 1]
```diff
@@ -23,7 +23,8 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        SimPrintsLibrary.init(MainActivity.this,"tZqJnw0ajK04LMYdZzyw","test_user");
+//        SimPrintsLibrary.init(MainActivity.this,"tZqJnw0ajK04LMYdZzyw","test_user");
+        SimPrintsLibrary.init(MainActivity.this,"fUBnpzDdbsCsMp0egCHB","global_module");
```
Comment (Andati) [inline sample/src/main/java/org/smartregister/simprint/sample/MainActivity.java]: It would be good to move this to local.properties

Comment (issyzac) [reply sample/src/main/java/org/smartregister/simprint/sample/MainActivity.java]: Okay, that makes sense. I have moved it already

Comment (issyzac) [reply sample/src/main/java/org/smartregister/simprint/sample/MainActivity.java]: I have also updated the sample app to include the identification piece.

**Review by Andati — APPROVED** [review]

**Pull request merged**